ledger/sandbox-common/src/test/lib/scala/platform/sandbox/cli/CommonCliSpecBase.scala

```diff
@@ -356,6 +356,18 @@ abstract class CommonCliSpecBase(
         _.copy(metricsReportingInterval = 90.seconds),
       )
     }
+
+    "parse error codes v2 flag" in {
+      checkOption(
+        Array("--use-error-codes-v2"),
+        _.copy(enableErrorCodesV2 = true),
+      )
+      checkOption(
+        Array(),
+        _.copy(enableErrorCodesV2 = false),
+      )
+    }
+
```

Comment:
I would check the default here as well, just like you have done for the app/Config

Reply:
I think the second `checkOption` is doing that